lib/flutter/flutter_sdk.dart

```diff
@@ -44,15 +46,17 @@ class FlutterSdkManager implements Disposable {
       .transform(new Debounce(new Duration(seconds: 1)))
       .listen((value) => _setSdkPath(value));
 
-    _disposables.add(atom.commands.add('atom-workspace', 'flutter:auto-locate-flutter-sdk', (_) {
-      new SdkLocationJob(this).schedule();
-    }));
-    _disposables.add(atom.commands.add('atom-workspace', 'flutter:show-flutter-sdk-info', (_) {
-      showInstallationInfo();
-    }));
-    _disposables.add(atom.commands.add('atom-workspace', 'flutter:version', (_) {
-      showVersionInfo();
-    }));
+    if (Flutter.hasFlutterPlugin()) {
```

Comment:
Landed PR to [show Dartino specific commands iff dartino plugin installed](https://github.com/dart-atom/dartlang/pull/984/commits/d1e3ffa5b01617a3bad808e90e8d53f069839d4f)